src/mesh/mechanical/plane/femMesh.hpp

```diff
@@ -81,6 +85,7 @@ class femMesh
     [[nodiscard]] List filter_dof_list(std::vector<Submesh> const& boundary_mesh) const;
 
 protected:
+    double generate_time_step;
```

Comment:
Spacing between this and the next member variable.  Maybe a little documentation on what it does too?